GitHubComments.ps1

```diff
@@ -181,7 +181,7 @@ function Get-GitHubComment
         'UriFragment' = $uriFragment
         'Description' = $description
         'AccessToken' = $AccessToken
-        'AcceptHeader' = (Get-MediaAcceptHeader -MediaType $MediaType)
+        'AcceptHeader' = (Get-MediaAcceptHeader -MediaType $MediaType -AcceptHeader 'application/vnd.github.squirrel-girl-preview')
```

Comment:
Seems like it would make sense to have a `$script`-level constant for `application/vnd.github.squirrel-girl-preview`, and then we can start making constants for others in a later change.